diff --git a/node_modules/next/dist/build/utils.js b/node_modules/next/dist/build/utils.js
index 29a2f1e..7431b93 100644
--- a/node_modules/next/dist/build/utils.js
+++ b/node_modules/next/dist/build/utils.js
@@ -577,7 +577,7 @@ async function isPageStatic({ dir, page, distDir, configFileName, httpAgentOptio
             hasStaticProps: false,
             hasServerProps: false,
             isNextImageImported: false,
-            appConfig: {}
+            appConfig: {revalidate: 0}
         };
     }
     await (0, _createincrementalcache.createIncrementalCache)({
